.github/pull_request_template.md

```diff
@@ -20,7 +20,9 @@
   - If so, please merge to a feature branch so documentation updates only go live upon official release.
 - [ ] This is planned work for an upcoming release.
   - If no, author or reviewer should account for this in a release plan, or describe why not below.
-<!-- Please add the planned version as a label on this PR -->
```

Comment:
Whitespace must have gotten lost when I auto-applied a suggestion from github UI, and I wanted to specify lets make use of "milestone" labels.